src/app/components/header/header.component.ts

```diff
@@ -29,15 +29,17 @@ export class HeaderComponent implements OnInit {
     this.authService.authSubject.subscribe((res: Partial<User>): void => {
       this.userName = !!res?.name ? res.name : '';
     });
-    this.modalWindowService.modalSubject.subscribe((result: boolean) => (this.isVisible = result));
+    this.modalWindowService.visible.subscribe((result: boolean) => (this.isVisible = result));
   }
 
   openModal(): void {
     this.modalWindowService.visible.next(true);
+    setTimeout(() => {
```

Comment:
The same

Reply:
> The same

because otherwise subscribe isn't watch the data. only log, without assign data to variable